clj/src/slipstream/ui/models/vms.clj

```diff
@@ -11,7 +11,7 @@
                     :measurement])
       (assoc        :username           (-> vm-metadata :user       not-empty))
       (assoc        :cloud-instance-id  (-> vm-metadata :instanceId not-empty))
-      (assoc        :run-uuid           (-> vm-metadata :runUuid    (s/replace "Unknown" "") not-empty))
```

Comment:
This must have been the case previously, as it's expected by the tests (see below). I ignored this behaviour was changed.
If so, please create an issue to remove this line if you think it's appropriate.

In the meantime, I think leaving this `replace` is not a show-stopper for the hotfix of this PR.